pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -215,7 +215,8 @@ func (o *MercuryTriggerService) process(timestamp int64) {
 }
 
 func wrapReports(reportList []mercury.FeedReport, eventID string, timestamp int64) (capabilities.CapabilityResponse, error) {
-	val, err := mercury.Codec{}.Wrap(reportList)
+	// MercuryTriggerService only handles reports submitted by the local node - no need to validate signatures.
```

Comment:
Is this guaranteed somehow? What's the downside of validating the signatures again?

Reply:
Yes, MercuryTriggerService exists only in the context of a local node, it doesn't process anything received remotely. 

This comment isn't even fully accurate because Wrap() wouldn't validate signatures anyway (only Unwrap() does it). Let me re-word it.